ir/state.cpp

```diff
@@ -146,48 +146,38 @@ static expr eq_except_padding(const Type &ty, const expr &e1, const expr &e2) {
   return result;
 }
 
-expr State::strip_undef_and_add_ub(const Value &val, const expr &e) {
-  if (isUndef(e)) {
```

Comment:
I changed the name of this function to avoid ambiguity because there was another isUndef() at globals.h.